pkg/virt-operator/install-strategy/validations_generated.go

```diff
@@ -0,0 +1,7068 @@
+package installstrategy
+
+var resources map[string]string = map[string]string{
```

Comment:
I think this generated file is actually kind of clever. It lets us maintain the ability to programatically build the CRDs the way we are now, yet still use the standard controller-gen logic to generate the schema on our behave. 

basically, it lets us go with the flow of how crd generation logic is being supported in the k8s ecosystem while still maintaining our current design patterns.

Reply:
I am opened to any suggestions. What do you think about `CRDsValidation`?